tikv-client/src/main/java/com/pingcap/tikv/columnar/TiChunkColumnVector.java

```diff
@@ -133,8 +133,8 @@ private long getTime(int rowId) {
       day = 1;
     }
     if (this.type instanceof DateType) {
-      // only return day from epoch
-      return LocalDate.of(year, month, day).toEpochDay();
+      LocalDate date = new LocalDate(year, month, day, null);
+      return date.toDate().getTime() / 24 / 3600 / 1000;
     } else if (type instanceof DateTimeType || type instanceof TimestampType) {
       // only return microsecond from epoch.
       Timestamp ts =
```

Comment:
is it ok to use Timestamp here?

Reply:
yes, we are always using jvm time zone.

Reply:
Discussed offline, it is ok to use Timestamp here.